spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/secretmanager/SecretPropertySourceTests.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.autoconfigure.secretmanager;
+
+import com.google.cloud.secretmanager.v1beta1.SecretVersionName;
+import org.junit.Test;
+
+import org.springframework.cloud.gcp.core.GcpProjectIdProvider;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SecretPropertySourceTests {
```

Comment:
The name doesn't match the class `SecretManagerPropertySource`.

Reply:
Done.